StoreKitUnitTests/TransactionsManagerSK2Tests.swift

```diff
@@ -41,30 +41,11 @@ class TransactionsManagerSK2Tests: StoreKitConfigTestCase {
     func testSK2CheckCustomerHasTransactionsWithOnePurchaseAsync() async throws {
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
 
-        try await simulateAnyPurchase()
+        try await self.simulateAnyPurchase()
 
         let receivedHasTransactionsValue = await transactionsManager.sk2CheckCustomerHasTransactions()
 
         expect(receivedHasTransactionsValue) == true
     }
 
 }
-
-private extension TransactionsManagerSK2Tests {
```

Comment:
This was duplicated 3 times now so I extracted it to `StoreKitTestHelpers`

Reply:
👏